### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-ru/content_questlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#289]
```
some translate, grammar, spelling and stylistic fixes.
```

Original issue reported on code.google.com by `e.solodo...@gmail.com` on 29 Dec 2011 at 8:18

Attachments:
- [content_questlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-289/comment-0/content_questlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thank you.
```

Original comment by `sdeva...@gmail.com` on 30 Dec 2011 at 7:44
- Changed state: **Accepted**
- Added labels: **Type-Translation**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:47
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
